### PR TITLE
cpu_rdrand: Add test case for rdrand

### PR DESCRIPTION
**Pull request by vivianQizhu**
id: 1874345
Signed-off-by: Qianqian Zhu <qizhu@redhat.com>

### CODE REVIEW

**Comment by vivianQizhu** [comment]
@nanliu-r Please help review, thanks.
 (1/1) Host_RHEL.m8.u3.product_rhel.ovmf.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.cpu_rdrand.q35: PASS (52.06 s)
 (1/1) Host_RHEL.m8.u3.product_rhel.qcow2.virtio_blk.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.cpu_rdrand: PASS (106.41 s)

File: deps/rdrand/rdrand.c [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+{
+    return (float)(rand())/RAND_MAX*(max - min) + min;
+}
+
+int main()
+{
+    srand((unsigned int)time(0));
+    printf("%f\n",randoms(-100.001, 100.001));
+    return 0;
+}
```
Comment (nanliu-r) [inline deps/rdrand/rdrand.c:15]: need one empty line, others look good to me.

Comment (vivianQizhu) [reply deps/rdrand/rdrand.c]: @nanliu-r Updated, thanks.

**Comment by nanliu-r** [comment]
Test pass, Thanks.
 (1/2) Host_RHEL.m8.u3.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.1.x86_64.io-github-autotest-qemu.cpu_rdrand: PASS (40.20 s)
 (2/2) Host_RHEL.m8.u3.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.Win10.x86_64.io-github-autotest-qemu.cpu_rdrand: PASS (89.89 s)

ACK.

**Comment by vivianQizhu** [comment]
Thanks @nanliu-r

**Pull request merged**